uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/contact-edit-form.js

```diff
@@ -15,7 +15,7 @@ module.exports = (function () {
       return new Promise(function (resolve) {
```

Comment:
is this even about a civihr page? isn't this for a core screen? if yes, it should be removed completely

Reply:
Right, this is**not** CiviHR screen, but a core screen. Removed.